test/distributed/_tensor/test_tensor_ops.py

```diff
@@ -229,6 +229,43 @@ def test_zeros_like(self):
         zeros_expected = torch.zeros(4, 8)
         self.assertEqual(zeros_expected, zeros_like_dt.to_local())
 
+    @with_comms
+    def test_stack(self):
```

Comment:
```
RuntimeError: shape '[2, 2]' is invalid for input of size 2
```
may need to skip the 2D part of the test unless we have 4 GPUs?

Reply:
done

Reply:
Just curious, does `test_tensor_ops.py` run on 4 GPUs in the periodic runner?

Reply:
I think if we change the dtensor code, we should have some inductor CI (4GPUs) running the tests https://github.com/pytorch/pytorch/blob/main/.github/labeler.yml#L36

And I think we have CPU only CI should run this with world size 4